### PR TITLE
build: publish@4.2.0

### PR DESCRIPTION
**Pull request by ZLY201**
Publish version 4.2.0

Expected to be released at 11.20.

**note: The last commit must be build: publish@{version}.**

### CODE REVIEW

**Comment by ZLY201** [comment]
`dev` branch locked.

**Comment by ZLY201** [comment]
Help to check the version and changelog. @noname0310

File: src/container/HashContainer/HashSet.ts [hunk 1]
```diff
@@ -1,55 +1,93 @@
...
+    if (this._node === this._header) {
+      throwIteratorAccessError();
+    }
+    return this._node._key;
+  }
+  copy() {
+    return new HashSetIterator(this._node, this._header, this.iteratorType);
+  }
+  // @ts-ignore
+  equals(iter: HashSetIterator<K, V>): boolean;
```
Comment (noname0310) [inline src/container/HashContainer/HashSet.ts:17]: Why do you use ts-ignore?

**Review by noname0310 — APPROVED** [review]

File: src/container/HashContainer/HashSet.ts (hunk 1 above)
Comment (ZLY201) [reply src/container/HashContainer/HashSet.ts:17]: TS tell me it must have a function body.

**Pull request merged**